contracts/BlastERC20RebasingYield.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {BlastNativeYield} from "./BlastNativeYield.sol";
+import {IERC20Rebasing, YieldMode} from "./interfaces/IERC20Rebasing.sol";
+
+/**
+ * @title BlastERC20RebasingYield
+ * @notice This contract is a base contract for inheriting functions to claim Blast WETH or USDB yield
+ * @author LooksRare protocol team (👀,💎)
+ */
+contract BlastERC20RebasingYield is BlastNativeYield {
+    address public immutable WETH;
+    address public immutable USDB;
+
+    /**
+     * @param _blast Blast precompile
+     * @param _blastPoints Blast points
+     * @param _blastPointsOperator Blast points operator
+     * @param _governor The address that’s allowed to claim the contract’s yield and gas
+     * @param _usdb USDB address
+     * @param _weth WETH address
+     */
+    constructor(
+        address _blast,
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _governor,
+        address _usdb,
+        address _weth
+    ) BlastNativeYield(_blast, _blastPoints, _blastPointsOperator, _governor) {
+        WETH = _weth;
+        USDB = _usdb;
+
+        IERC20Rebasing(_weth).configure(YieldMode.CLAIMABLE);
```

Comment:
Just to confirm, for all weth and usdb holding contracts, the yield mode will always be claimable, never automatic?

Reply:
I think it's fine to leave it as the default and if we want to change it we can do it after - we've never had any contracts with a claim mode that's not CLAIMABLE